File: java/questao4.java
import java.util.Scanner;

public class questao4 {

    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);
        System.out.print("Digite a sua idade: ");
        int n1 = scan.nextInt();
        if (n1  >= 18){
            System.out.print("Você é maior de idade.");
        }else{
            System.out.print("Você é menor de idade.");
        }
}
}

File: questao4.c
#include<stdio.h>


void main()
{
    int idade;
    printf("Digite a sua idade: ");
    scanf("%d", &idade);

    if(idade>=18){
        printf("Voce e maior de idade");
    }   else{
        printf("Voce nao e maior de idade");
    }
    
}